project/home/templates/home/invite_friends.html

```diff
@@ -0,0 +1,17 @@
+{% extends "base.html" %}
+{% load wagtailcore_tags %}
+
+{% block content %}
+    <h1>{{ page.title }}</h1>
+
+    {{ page.intro | richtext }}
+
+    <form action="." method="POST">
+        {% csrf_token %}
+
+        {{ form.as_p }}
+
+        <p><input type="submit" value="Send Invitation Now!"></p>
```

Comment:
Let's use [Bootstrap button classes](https://getbootstrap.com/docs/4.0/components/buttons/) so this renders as a button.

```suggestion
        <p><input type="submit" class="btn btn-primary" value="Send Invitation Now!"></p>
```